### PR TITLE
Delete a request from admin request group view

### PR DESCRIPTION
**Pull request by kevin-wangg**

### CODE REVIEW

File: client/src/components/molecules/RequestsTable.tsx [hunk 1]
```diff
@@ -56,8 +66,20 @@ const RequestsTable: FunctionComponent<Props> = (props: Props) => {
...
      //console.log(sortedRequests);
-
+    
+    const [mutateDeleteRequest] = useMutation(softDeleteRequest)   
     const [mutateRequest] = useMutation(updateRequest);
+    const onSoftDeleteRequest = (index: number) => {
+        const requestsCopy = requests.slice()
+        const req = {...requestsCopy[index]}
+        req.deleted = true
+        console.log(req)
```
Comment (meganniu) [inline client/src/components/molecules/RequestsTable.tsx]: console.log

Comment (kevin-wangg) [reply client/src/components/molecules/RequestsTable.tsx]: Removed.

File: client/src/components/molecules/RequestsTable.tsx [hunk 2]
```diff
@@ -56,8 +66,20 @@ const RequestsTable: FunctionComponent<Props> = (props: Props) => {
       }, []);
       
      //console.log(sortedRequests);
```
Comment (meganniu) [inline client/src/components/molecules/RequestsTable.tsx]: lol could u also help remove this (even tho it's not urs)

File: client/src/components/molecules/RequestsTable.tsx [hunk 3]
```diff
@@ -21,6 +22,15 @@ const RequestsTable: FunctionComponent<Props> = (props: Props) => {
...
+    mutation deleteRequest($id: ID) {
+        softDeleteRequest(id: $id) {
+            id
+            success
+            message
+        }
+    }`
+
 
     const [requests, setRequests] = useState(props.requests);
```
Comment (meganniu) [inline client/src/components/molecules/RequestsTable.tsx]: Would it be possible to just initialize requests as 
```
const [requests, setRequests] = useState(props.requests.filter((request)=> request.deleted === false));
```

So then getting the number of request is just `requests.length`

**Review by meganniu — APPROVED** [review]
Left one suggestion, but otherwise looks good.
thanks kevin!!!

File: client/src/components/molecules/RequestsTable.tsx (hunk 2 above)
Comment (kevin-wangg) [reply client/src/components/molecules/RequestsTable.tsx]: Removed.

File: client/src/components/molecules/RequestsTable.tsx (hunk 3 above)
Comment (kevin-wangg) [reply client/src/components/molecules/RequestsTable.tsx]: Updated.

**Review by chamod-gamage — APPROVED** [review]
lgtm!

**Pull request merged**